index.html

```diff
@@ -3,6 +3,18 @@
 key: news
 ---
 
+<div>
+  <div class="bp3-card card__title">
+    <h3 class="card__title__text">APIO 2021 Rules</h3>
+    <p class="card__title__right">15 March 2021</p>
+  </div>
+  <div class="bp3-card card__content">
+    <p>
+      Some changes are made from APIO 2020 rules. <br/>
+      More details can be found in the <a href="rules">APIO 2021 Competition Rules</a> page.
```

Comment:
`Several information in the competition rules have been updated. Please check the updated <a href="rules">APIO 2021 Competition Rules</a>.`